code/modules/crafting/tailoring.dm

```diff
@@ -212,12 +212,29 @@
 	reqs = list(/obj/item/clothing/glasses/hud/janitor/sunglasses = 1)
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/hudsunmeson
+	name = "Meson HUD sunglasses"
+	result = list(/obj/item/clothing/glasses/meson/sunglasses)
+	time = 20
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/meson = 1,
+				/obj/item/clothing/glasses/sunglasses = 1,
+				/obj/item/stack/cable_coil = 5)
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/hudsunmesonremoval
+	name = "Meson HUD sunglasses removal"
+	result = list(/obj/item/clothing/glasses/sunglasses, /obj/item/clothing/glasses/meson)
+	time = 20
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/meson/sunglasses = 1)
+	category = CAT_CLOTHING
 /datum/crafting_recipe/beergoggles
 	name = "Sunscanners"
 	result = list(/obj/item/clothing/glasses/sunglasses/reagent)
 	time = 20
 	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
-	reqs = list(/obj/item/clothing/glasses/science = 1,
+	reqs = list(/obj/item/clothing/glasses/meson = 1,
```

Comment:
🤨

Reply:
Why do you want me to make the bartender's sunscanners craftable with mesons?

Reply:
I'm an idiot hold on.